### PR TITLE
Rework the container development environment.

### LINKED ISSUE #264: Simple container development environment
Currently the docker-compose setup requires running several command manually.

Ideally it should be a one-command setup. This would ease up onboarding new contributors.

### PR DESCRIPTION
**Pull request by Iizuki**
`$ podman-compose -f containers/composefile.yaml up` now sets up the whole thing and watches for changes BUT the project is run using spago, which isn't ideal.

Let's figure out something else in it's place.

fixes #264

### CODE REVIEW

**Comment by easafe** [comment]
@zeved 

`spago run` can only start the server part. It won't restart it in case of changes. It also doesn't work for the client side. The files need to be bundled to run on the browser

File: containers/merochat.containerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+RUN curl -fsSL https://apt.cli.rs/pubkey.asc >> /usr/share/keyrings/rust-tools.asc
+ADD https://apt.cli.rs/rust-tools.list /etc/apt/sources.list.d/rust-tools.list
+
+RUN apt-get update && \
+    apt-get install --assume-yes watchexec-cli && \
+    rm -rf /var/lib/apt/lists/*
+
+# Start the application and watch for changes.
+# https://github.com/purescript/spago?tab=readme-ov-file#watch-mode
+CMD [ "watchexec", "--exts=purs,js,yaml", "--restart", "--", "spago", "run", "--main", "Server.Main" ]
```
Comment (Iizuki) [inline containers/merochat.containerfile]: This spago command should probably be replaced with something more nuanced @easafe

Comment (easafe) [reply containers/merochat.containerfile]: I think if you just include webpack then it is fine

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,13 +16,21 @@ Don't hesitate to [contact](https://github.com/easafe) if you need help, or want
...
+
+### Containers
+
 You can run MeroChat locally with podman or docker.
 
-1. Run `sudo docker-compose -f containers/composefile.yaml up` to spin up the containers
+Just run `podman-compose -f containers/composefile.yaml up` to spin up the containers.
+Your very own MeroChat is now accessible at http://localhost:8000, and will be rebuilt when you make changes to the code. This isn't instantaneous though. Keep an eye at the container runtime output.
+
+Finally tear the containers down with `ctr+c` followed by `podman-compose -f containers/composefile.yaml down`
```
Comment (easafe) [inline CONTRIBUTING.md]: Should we mention docker here too?

Comment (Iizuki) [reply CONTRIBUTING.md]: We can do that yes, though I  hold that podman is better 😏

Comment (easafe) [reply CONTRIBUTING.md]: you are the boss

**Pull request merged**